pkg/scheduler/framework/session_plugins.go

```diff
@@ -503,6 +507,30 @@ func (ssn *Session) ReservedNodes() {
 	}
 }
 
+// Victims returns the victims for rescheduling
+func (ssn *Session) Victims(tasks []*api.TaskInfo) map[*api.TaskInfo]bool {
```

Comment:
This function should be part of action instead of this plugin.

Reply:
`Victims` get registered functions from `ssn.reschedulingFns` and then execute them. I think it is better add it in this PR.

Reply:
One PR for framework (shuffle action), one pr for implementation (rescheduling plugin).
`Victims` should be also used by TDM plugin.

Reply:
> One PR for framework (shuffle action), one pr for implementation (rescheduling plugin).
> `Victims` should be also used by TDM plugin.

I get you. It's OK for me. Let me make a update.